frontend/src/tests/ddah-tests.js

```diff
@@ -67,7 +67,7 @@ export function ddahTests(api) {
             ],
         };
 
-        let resp = await apiPOST(`/admin/ddahs`, newDdah);
+        const resp = await apiPOST(`/admin/ddahs`, newDdah);
```

Comment:
I see that you have changed the resp object to be a constant every time you make an API request. Even though this is not incorrect, it might be more beneficial to make resp a global variable declared with let and used in every test. This will also allow you to make more than 1 API request within a single test :).

Reply:
I believe that was my local linter that changed these :smile:

Usually, I try to stick to using `const` whenever possible, and `let` only if I know the variable will change. This helps to avoid some cases where I expect that variable to hold some value before it's reassigned in that test. Kind of similar to [this thread](https://softwareengineering.stackexchange.com/questions/278652/how-much-should-i-be-using-let-vs-const-in-es6).

I believe it's also a common practice / style decision made in many es6 codebases, just like using global variables is a code smell usually (obviously it might be warranted for some cases like the test suite we have). Of course, it's a style matter though!

Reply:
Let's put them back as `let`. This change doesn't have anything to do with this PR